main.c

```diff
@@ -65,10 +65,7 @@ static void on_on_ecall(struct riscv_t *rv)
     syscall_handler(rv);
 }
 
-static void on_on_ebreak(struct riscv_t *rv)
-{
-    rv_halt(rv);
-}
+static void on_on_ebreak(struct riscv_t *rv) {}
```

Comment:
Avoid unrelated changes!

Reply:
The on_break function is not use after I solve the c.ebreak test.
- I treat the ebreak as the trap "break point"(in privilege specification) that pass the **c.ebreak** compliance test.
- Both(before/after modified) are **no pass** ebreak test in privilege mode.The I/M/Zifenci instruction test(sammer1107's compliance test passed) have **not** the "ebreak" test. 

Should I keep or remove related function?
- In the non-privilege mode the ebreak will let "the system back to the debug environment" But I don't know the detail in implementation
- The "on_ebreak" function let the system halt and terminate.(my understanding)

Reply:
The change against `on_on_ebreak` should appear in another pull request which addresses in the compliance and architecture test.